src/applications/vaos/appointment-list/components/cards/confirmed/ConfirmedCommunityCareLocation.jsx

```diff
@@ -10,21 +10,36 @@ export default function ConfirmedCommunityCareLocation({ appointment }) {
     return null;
   }
 
+  const locationMessage =
+    'This appointment is scheduled with a community care provider. Please do not report to your local VA facility. If you have questions please contact your facility community care staff at ';
+
   if (!location.address) {
-    return (
-      <>
-        <h4 className="vaos-appts__block-label">{location.name}</h4>
-        <div>
-          This appointment is scheduled with a community care provider. Please
-          do not report to your local VA facility. If you have questions, please
-          contact your facility community care staff at{' '}
-          <a href="/find-locations" target="_blank" rel="noopener noreferrer">
-            your local VA.
-          </a>
-          <br />
-        </div>
-      </>
-    );
+    if (!location.name) {
```

Comment:
This seems a little bit difficult to read now. When I suggested the change originally, I was thinking of something like changing the line with the h4 to `{!!location.name && <h4 className="vaos-appts__block-label">{location.name}</h4>}`

Reply:
Brilliant! change made.